packages/gasket-helper-intl/lib/index.js

```diff
@@ -149,6 +149,10 @@ function LocaleUtils(config) {
     const mappedLocale = localesMap && localesMap[locale] || locale;
     let fallbackLocale = mappedLocale;
 
+    if (locales && !locales.includes(locale)) {
```

Comment:
Shouldn't this be checking if `mappedLocale` is in `locales`?

Reply:
Yep! Great catch!